apps/fulltextsearch.sh

```diff
@@ -27,7 +29,7 @@ root_check
 lowest_compatible_nc 18
 
 # Check if Full Text Search is already installed
-if ! does_this_docker_exist "$nc_fts" || ! is_app_installed fulltextsearch
+if ! does_this_docker_exist "$nc_fts" || ! does_this_docker_exist "$opens_fts" || ! is_app_installed fulltextsearch
```

Comment:
```suggestion
if ! does_this_docker_exist "$nc_fts" && ! does_this_docker_exist "$opens_fts" && ! is_app_installed fulltextsearch
```

Reply:
Explanation: we only want to show the install_popup if neither of those containers/app was found. If it was, we want to show the option for reinstallation or uninstallation.

Reply:
yeah, ok.